tasks/preparation-provisioning.yaml

```
@@ -0,0 +1,56 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: preparation-provisioning
+spec:
+  params:
+   - name : BONFIRE_IMAGE
+     type: string
+     description: The container Bonfire image to use for the tekton tasks
+   - name: CUSTOM_IMAGE
+     type: string
+     description: The container image to use for the provisioning setup task
+     default: quay.io/envision/provisioning-cicd:latest
+   - name: NS_REQUESTER
+     type: string
+     description: The name of the person/pipeline that requested the namespace
+   - name: EPHEMERAL_ENV_PROVIDER_SECRET
+     type: string
+     default: ephemeral-env-provider
+     description: "Secret for connecting to ephemeral env provider cluster"
+   - name: AWS_ACCOUNT_ID
```

I'd name that something different, this is bit confusing
```suggestion
   - name: AWS_ACCOUNT_ID_SECRET
```